cpp/cgi_progs/zotero_cgi.cc

```diff
@@ -578,9 +579,9 @@ bool ProcessShowQASubActionAdd(const std::multimap<std::string, std::string> &cg
     if (add_type == "global")
         journal_id_to_insert = "NULL";
 
-    db_connection->queryOrDie("INSERT INTO metadata_presence_tracer (zeder_journal_id, marc_field_tag, field_presence) "
-                              " VALUES (" + journal_id_to_insert + ", " + db_connection->escapeAndQuoteString(add_tag) + ", "
-                              " " + db_connection->escapeAndQuoteString(add_presence) + ")");
+    db_connection->queryOrDie("INSERT INTO metadata_presence_tracer (journal_id, marc_field_tag, marc_subfield_code, field_presence) "
+                              " VALUES (" + journal_id_to_insert + ", " + db_connection->escapeAndQuoteString(add_tag) + ", '"
```

Comment:
escapeAndQuoteString could be used here as well (consistency)

Reply:
I don't think that's any more readable.